connector/connect/common/src/main/protobuf/spark/connect/relations.proto

```diff
@@ -394,6 +395,18 @@ message CachedLocalRelation {
   string hash = 3;
 }
 
+// Represents a DataFrame that has been cached on server.
+message CachedDataFrame {
+  // (Required) An identifier of the user which cached the dataframe
+  string userId = 1;
+
+  // (Required) An identifier of the Spark session in which the dataframe is cached
+  string sessionId = 2;
+
+  // (Required) A key represents the id of the cached dataframe
+  string key = 3;
```

Comment:
Better to rename this as `id`, `relationId`, or `remoteId`.

Reply:
update to relationId